Dockerfile

```diff
@@ -25,13 +25,19 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
     mv tini /usr/local/bin/tini && \
     chmod +x /usr/local/bin/tini
 
+# Install Mysql regex UDFs
+RUN wget --quiet https://github.com/mysqludf/lib_mysqludf_preg/archive/testing.zip && \
+    mkdir lib_mysqludf_preg && \
+    unzip testing.zip -d lib_mysqludf_preg && \
+    cd lib_mysqludf_preg && \
+    ./configure && make install && make installdb && make test;
```

Comment:
Ouch, those are file permission errors. As in, MySQL does not have permission to write to your datadir. This was pretty common with older versions of the MySQL container, and solved in a workable way earlier this year.

Looking at the [build history for the docker-mysql-udf-preg container](docker-mysql-udf-preg), the last build was a year ago, probably before the MySQL container worked out the kinks.

Options to move forward are, presented on a range of correct -> fast;

1) Poke the owner, see if he could push a new build. 
2) Fork the repo, and do it ourselves. (I'd be happy to do this)
3) Create a local Dockerfile for MySQL, copy-pasta the pertinent line, and have `docker-compose` manage the build for us.